_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -41,9 +41,29 @@ PUT /my-index1
   }
 }
 ````
+{% include copy-curl.html %}
 
 In segment replication, the primary shard is usually generating more network traffic than the replicas because it copies segment files to the replicas. Thus, it's beneficial to distribute primary shards equally between the nodes. To ensure balanced primary shard distribution, set the dynamic `cluster.routing.allocation.balance.prefer_primary` setting to `true`. For more information, see [Cluster settings]({{site.url}}{{site.baseurl}}/api-reference/cluster-api/cluster-settings/).
 
+Please note that segment replication currently does not support `wait_for requests` in refresh API.
+{: .warning }
+
+For the best performance, we recommend enabling the following settings:
+
+1. [Segment replication back-pressure]({{site.url}}{{site.baseurl}}tuning-your-cluster/availability-and-recovery/segment-replication/backpressure/). 
+2. Balanced primary shard allocation. 
+
+```json
+curl -X PUT "$host/_cluster/settings?pretty" -H 'Content-Type: application/json' -d'
+  {
+    "persistent": {
+    "cluster.routing.allocation.balance.prefer_primary": true,
+    "segrep.pressure.enabled": true
+   }
+  }
+```
+{% include copy-curl.html %}
+
```

Comment:
Remove the following text as per SME:
```
The tests were run with the following settings:
- `indices.recovery.max_bytes_per_sec`: 10gb
- `indices.recovery.max_concurrent_file_chunks`: 5```

Reply:
Also, remove the mention of stackoverflow data because these results are only for the nyc_taxi dataset. And since we're not providing all results, we can remove the word "both". Alternatively, you can include the whole table here with all results, as will be done in the blog.